test/test_cpp_api_parity.py

```diff
@@ -1,687 +1,64 @@
+MESSAGE_HOW_TO_FIX_CPP_PARITY_TEST_FAILURE = '''
+What should I do when C++ API parity test is failing?
+
+- If you are changing the implementation of an existing `torch.nn` module / functional:
+Answer: Ideally you should also change the C++ API implementation for that module / functional
+(you can start by searching for the module / functional name in `torch/csrc/api/` folder).
+
+- If you are adding a new test params dict for an existing `torch.nn` module / functional:
```

Comment:
What is a "new test params dict"?

Reply:
Oh, I see. Maybe call them "ModuleTest" or NN module test or link the dicts in common_nn.py?